src/plugins/orangehrmRecruitmentPlugin/entity/Decorator/CandidateHistoryDecorator.php

```diff
@@ -86,6 +88,13 @@ public function setPerformedBy(int $id): void
     public function getCandidateHistoryAction(): string
     {
         $actionId = $this->candidateHistory->getAction();
-        return ucwords(strtolower(CandidateService::STATUS_MAP[$actionId]));
+        $candidateHistoryMap = array_replace(CandidateService::STATUS_MAP, CandidateService::OTHER_ACTIONS_MAP);
+        return ucwords(strtolower($candidateHistoryMap[$actionId]));
+    }
+
+    public function getPerformedDate(): string
+    {
+        $date = $this->candidateHistory->getPerformedDate();
+        return $this->getDateTimeHelper()->formatDateTimeToYmd($date);
```

Comment:
```suggestion
        return $this->getDateTimeHelper()->formatDate($date);
```
check this method and continue it